taskcat/_config.py

```diff
@@ -374,3 +476,131 @@ def _harvest_env_vars(self, env_vars):
                 elif value.lower() in ["true", "false"]:
                     value = value.lower() == "true"
                 self.env_vars[key] = value
+
+
+class S3Bucket:
```

Comment:
in the interest of keeping this file small (well, to be fair, lets call it `less humungous`) can we move this out into it's own file.